src/activities/gletters/gletters.js

```diff
@@ -99,7 +99,7 @@ function initLevel() {
         // If "sublevels" length is not set in wordlist, use the words length
         maxSubLevel = level.words.length
     }
-    items.score.numberOfSubLevels = maxSubLevel;
+    items.score.numberOfSubLevels = maxSubLevel + 1;
```

Comment:
On Tue, Feb 17, 2015 at 10:34 PM, Bruno Coudoin
<notifications@github.com> wrote:
>
> In src/activities/gletters/gletters.js:
>
> > @@ -99,7 +99,7 @@ function initLevel() {
> >          // If "sublevels" length is not set in wordlist, use the words length
> >          maxSubLevel = level.words.length
> >      }
> > -    items.score.numberOfSubLevels = maxSubLevel;
> > +    items.score.numberOfSubLevels = maxSubLevel + 1;
>
> I don't agree here. It currently works fine, for 5 data we go from 1/1 to 5/5

I found it confusing to still have data left to type when it showed
5/5, I assumed
I had typed everything when reaching 5/5 but there was one character left.

I always viewed the sublevel as a score count, which created the confusion.
Starting the count on zero (0) would better match this.